### PR TITLE
Update 2.7 branch to 2.7.8 to fix CVE-2023-28756

### PR DESCRIPTION
**Pull request by kevinjqiu**
Update 2.7 branch to 2.7.8 to fix [CVE-2023-28756](https://www.ruby-lang.org/en/news/2023/03/30/redos-in-time-cve-2023-28756/)

### CODE REVIEW

**Comment by ryanbourdais** [comment]
Hi @kevinjqiu  Thank you for the PR.

As mentioned in the readme, edits to this image should be applied to the [Dockerfile.template](https://github.com/CircleCI-Public/cimg-ruby/blob/main/Dockerfile.template) file. Then, during the next release the changes will be published. We don't edit generated Dockerfiles directly.

Would you be able to make these changes?

**Comment by kevinjqiu** [comment]
@ryanbourdais Thanks for the pointer. Changes made.

File: 2.7/Dockerfile [hunk 1]
```diff
@@ -6,12 +6,11 @@
 # specific reason to use a different one. This means January, April, July, or
 # October.
 
-# NOTE: This uses November due to an OpenSSL CVE fix. We will resume using quarterly versions of cimg-base in Q1
-FROM cimg/base:2022.11-20.04
+FROM cimg/base:2023.01
```
Comment (akinomaeni) [inline 2.7/Dockerfile:9]: As noted in #116, we need to use ubuntu 20.04.
https://github.com/CircleCI-Public/cimg-ruby/pull/116/files#diff-8d7a21b017921bb88eaf71656b7b5767203db16e8126fc1e5ad2a9ba0bc542f5R9

Comment (ryanbourdais) [inline 2.7/Dockerfile:9]: @akinomaeni is correct, I just checked and this build fails currently

**Review by ryanbourdais — CHANGES_REQUESTED** [review]

**Review by ryanbourdais — APPROVED** [review]
It now builds this image successfully on the platform so it should be good to go, thanks again for contributing!

**Pull request merged**